src/Arcus.Templates.Tests.Integration/Authentication/v1/SharedAccessKeyAuthenticationOptionTests.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Arcus.Templates.Tests.Integration.Fixture;
+using Xunit;
+using Xunit.Abstractions;
+using static Arcus.Templates.Tests.Integration.Fixture.InMemorySecretProvider;
+
+namespace Arcus.Templates.Tests.Integration.Authentication.v1
```

Comment:
Can we add a similar test for Swagger UI?

- Spin up in debug, is able to navigate to Swagger UI
- Spin up in debug, is able to navigate to Swagger docs
- Spin up in release, is not able to navigate to Swagger UI
- Spin up in release, is not able to navigate to Swagger docs

Reply:
Great minds think alike

Reply:
But maybe not in this PR. Could you create an issue for this? Thanks.

Reply:
Done (#43)